airflow/www/views.py

```diff
@@ -2620,6 +2620,36 @@ def conf(self):
                 table=table)
 
 
+class ErrorLogView(wwwutils.SuperUserMixin, BaseView):
+    @expose('/')
+    def error_log(self):
+        raw = request.args.get('raw') == "true"
+        title = "Airflow Error Log"
+        log_base = os.path.expanduser(conf.get('core', 'BASE_LOG_FOLDER'))
+        error_log_path = os.path.normpath(log_base + "/error.log")
```

Comment:
This file is produced by the logging updates in PR #1921 that's mentioned above. This setup that outputs to this file is unrelated to stderr.